doc/bliss.xml

```diff
@@ -11,83 +12,58 @@
 <#GAPDoc Label="AutomorphismGroupDigraph">
 <ManSection>
   <Attr Name="AutomorphismGroup" Label="for a digraph" Arg="digraph"/>
-  <Oper Name="AutomorphismGroup" Label="for a digraph and a homogeneous list"
-    Arg="digraph, colors"/>
-  <Returns>A permutation group.</Returns>
+  <Returns>A permutation group, or a direct product of permutation
+    groups.</Returns>
   <Description>
-    If <A>digraph</A> is a digraph without multiple edges, then this function
-    returns the automorphism group of <A>digraph</A>, as a group of
-    permutations on the vertices of <A>digraph</A>. <P/>
+    If <A>digraph</A> is a digraph, then this attribute returns the group of
+    automorphisms of <A>digraph</A>.  An <E>automorphism</E> of <A>digraph</A>
+    is an isomorphism from <A>digraph</A> to itself. See <Ref
+      Oper="IsomorphismDigraphs" Label="for digraphs" /> for more information
+    about isomorphisms of digraphs. <P/>
 
-    If the <A>colors</A> argument is specified, then the group will consist of
-    only those automorphisms which respect the given colouring.  The colouring
-    <A>colors</A> can be in one of two forms: <P/>
+    The form in which the automorphism group is returned depends on whether
+    <A>digraph</A> has multiple edges; see <Ref Prop="IsMultiDigraph" />. <P/>
 
     <List>
+      <Mark>for a digraph without multiple edges</Mark>
       <Item>
-        A list of positive integers of size the number of vertices of
-        <A>digraph</A>, where <A>colors</A><C>[i]</C> is the colour of vertex
-        <C>i</C>.
+        If <A>digraph</A> has no multiple edges, then the automorphism group is
+        returned as a group of permutations on the vertices of <A>digraph</A>.
       </Item>
+      <Mark>for a multidigraph</Mark>
       <Item>
-        A list of lists, such that <A>colors</A><C>[i]</C> is a list of all
-        vertices with colour <C>i</C>.
+        If <A>digraph</A> is a multidigraph, then the automorphism group is
+        a group of permutations on the vertices and edges of <A>digraph</A>.
+        <P/>
+
+        For convenience, the group is returned as the direct product of the
+        group of automorphisms of the vertices of <A>digraph</A> with the
+        stabiliser of the vertices in the automorphism group of the edges. These
+        two groups can be accessed using the operation <Ref Oper="Projection"
+          Label="for a domain and a positive integer" BookName="ref"/>, with the
+        second argument being <C>1</C> or <C>2</C>, respectively. <P/>
```

Comment:
This is really good, can we say which projection corresponds to which action, and actually give an example? This would be most helpful.